src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -10,6 +10,8 @@
     // text displayed during feedback submission respectively.
     SELF(true, true, false, "Feedback session creator (i.e., me)", "Giver (Self feedback)", ""),
     STUDENTS(true, true, true, "Students in this course", "Other students in the course", "Other students in the course"),
+    STUDENTS_EXCLUDING_SELF(false, true, true, "Students in this course", "Other students in the course",
+            "Other students in the course"),
```

Comment:
The text displayed here is identical to the `STUDENTS` option, and that will be confusing to users.

Reply:
I'm not too sure what else to write for the text; it seems suitable for it's purpose. On another note, this new addition is not used in the feedback path so it won't be shown to the users.

Reply:
@Haozhe321 you'd have noticed by now that it really is not the case. ([example](#diff-b3cdf5a28c730589262ac1b1a1db2720R1747))

Reply:
The link brings me back to the pull request. Is it supposed to be an image or a code snippet?

Reply:
![image](https://user-images.githubusercontent.com/26651556/36383613-a06c27dc-15c7-11e8-8d0a-87eccd140071.png)
Is it this one? 
Should I change to "Other students in the course (excluding self)"? But other students kind of already means excluding self.